.github/workflows/check-pull-request.yaml

```
@@ -34,7 +34,37 @@ jobs:
-        go-version: 1.17
+        go-version: 1.25.10
 
     - name: Run the tests
       run: make tests
+
+  verify-model:
+    name: Verify model
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout the source
+      uses: actions/checkout@v2
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)